test/unit/object/test_file.py

```diff
@@ -278,3 +278,47 @@ def test_get_shared_link_download_url(
         params=None,
     )
     assert url == test_url
+
+
+def test_get_comments(test_file, mock_box_session):
+    expected_url = test_file.get_url('comments')
+    mock_box_session.get.return_value.json.return_value = {
+        'total_count': 2,
+        'offset': 0,
+        'limit': 100,
+        'entries': [
+            {
+                'type': 'comment',
+                'id': '1',
+                'message': 'Foo'
+            },
+            {
+                'type': 'comment',
+                'id': '2',
+                'message': 'Bar'
+            }
+        ]
+    }
+    test_file.get_comments().next()
```

Comment:
We can do more than this. Why not assert that we get two comments back with the expected comments?